cmd/register/examples.go

```diff
@@ -19,15 +19,18 @@ Registers all latest flytesnacks example
 
 Usage
 `
+	archive = true
+)
+
+var (
 	githubOrg        = "flyteorg"
 	githubRepository = "flytesnacks"
-	archive          = true
 	snackReleaseURL  = "https://github.com/flyteorg/flytesnacks/releases/download/%s/flytesnacks-%s.tgz"
 	flyteManifest    = "https://github.com/flyteorg/flytesnacks/releases/download/%s/flyte_tests_manifest.json"
 )
 
 func registerExamplesFunc(ctx context.Context, args []string, cmdCtx cmdCore.CommandContext) error {
-	flytesnacks, tag, err := getFlyteTestManifest()
+	flytesnacks, tag, err := getFlyteTestManifest(githubOrg, githubRepository)
```

Comment:
Can we convert these to const githubOrg, githubRepository

Reply:
need variable of unit test